mixer/cmd/repos.go

```diff
@@ -113,12 +114,18 @@ func runAddRepo(cmd *cobra.Command, args []string) {
 	if err != nil {
 		fail(err)
 	}
-
-	err = b.AddRepo(args[0], args[1])
+	u, err := url.Parse(args[1])
+	if err != nil {
+		fail(err)
+	}
+	if u.Scheme == "" {
+		u.Scheme = "file"
+	}
+	err = b.AddRepo(args[0], u.String())
```

Comment:
As I mentioned in the previous comment, this logic can be removed from here and can just be in `AddRepo()`

Reply:
As you mentioned, since AddRepo() is called only if repo does not exist, this review comment can be ignored.